modules/ppcp-order-tracking/src/OrderTrackingModule.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * The order tracking module.
+ *
+ * @package WooCommerce\PayPalCommerce\OrderTracking
+ */
+
+declare(strict_types=1);
+
+namespace WooCommerce\PayPalCommerce\OrderTracking;
+
+use Dhii\Container\ServiceProvider;
+use Dhii\Modular\Module\ModuleInterface;
+use Exception;
+use Interop\Container\ServiceProviderInterface;
+use Psr\Container\ContainerInterface;
+use Psr\Log\LoggerInterface;
+use WC_Order;
+use WooCommerce\PayPalCommerce\OrderTracking\Assets\OrderEditPageAssets;
+use WooCommerce\PayPalCommerce\OrderTracking\Endpoint\OrderTrackingEndpoint;
+use WooCommerce\PayPalCommerce\WcGateway\Exception\NotFoundException;
+use WooCommerce\PayPalCommerce\WcGateway\Helper\PayUponInvoiceHelper;
+use WooCommerce\PayPalCommerce\WcGateway\Settings\Settings;
+use WooCommerce\PayPalCommerce\WcGateway\Settings\SettingsListener;
+
+/**
+ * Class OrderTrackingModule
+ */
+class OrderTrackingModule implements ModuleInterface {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function setup(): ServiceProviderInterface {
+		return new ServiceProvider(
+			require __DIR__ . '/../services.php',
+			require __DIR__ . '/../extensions.php'
+		);
+	}
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @param ContainerInterface $c A services container instance.
+	 * @throws NotFoundException
+	 */
+	public function run( ContainerInterface $c ): void {
+		/**
+		 * The Settings.
+		 *
+		 * @var Settings $settings
+		 */
+		$settings = $c->get( 'wcgateway.settings' );
+
+		/**
+		 * The PUI helper.
+		 *
+		 * @var PayUponInvoiceHelper $pui_helper
+		 */
+		$pui_helper = $c->get( 'wcgateway.pay-upon-invoice-helper' );
```

Comment:
You can use `assert($x instanceof X)` instead of these huge phpdocs.

Reply:
done. 9ea71be37bb583e2de331ec1b8ea222c28629e91